packages/bot-web-ui/src/constants/bot-contents.ts

```diff
@@ -14,5 +14,3 @@ export const DASHBOARD_TABS: TTabsTitleProps = Object.freeze({
     CHART: 3,
     TUTORIAL: 4,
 });
-
-export const MAX_STRATEGIES = 10;
```

Comment:
@rupato-deriv are you sure that we need all strategies which had the user for example during a year? I think it will be heavy and unnecessary

Reply:
<img width="995" alt="image" src="https://user-images.githubusercontent.com/97010868/208622525-ccaac0f9-4397-42de-9c2c-5e4a9a945e69.png">
@maryiafrantsava-binary this limit is handled now in localstorage.js file which is is bot skeleton